src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js

```diff
@@ -221,7 +221,7 @@ describe('when moving to other profile sections', () => {
 });
 
 describe('Modals on the personal information and content page when they error', () => {
-  it('should focus on the close notification and the save button when the error is closed', () => {
```

Comment:
oh cool, it's nice you could get rid of all the focus management code related to the old error alert having a `close` button 👍